### ISSUE TITLE
Accessibility: missing labels on shop page orderby select box

### ISSUE THREAD

**Issue by madeincosmos** [woocommerce/woocommerce#22679]
**Is your feature request related to a problem? Please describe.**
The select element on the shop / category pages doesn't have either a label or an aria tag. This makes it difficult to use for people accessing the site via a screen reader.

**Describe the solution you'd like**
Add an aria label tag for the `orderby` select box

![Screenshot](http://cld.wthms.co/fLdhOy+)
Screenshot: http://cld.wthms.co/fLdhOy

**Comment by claudiulodro** [comment]
#22683
